.github/workflows/main.yml

```diff
@@ -162,12 +162,14 @@ jobs:
       timeout-minutes: 20
       run: |
        scons -j 2 package BUILD_TYPE=${{ matrix.buildType }} OPTIONS=.github/workflows/main/sconsOptions
+      if: matrix.publish
```

Comment:
Ah, so this is why it's beneficial to omit publishing for debug builds - to avoid the packaging overhead and make the debug runs finish quicker? Do you know why the packaging is so much slower for a debug build?